src/hooks/useTokenMaps.ts

```diff
@@ -0,0 +1,43 @@
+import { BasicTokensMap, PoolName, TokenToPoolsMap } from "../constants"
+
+import { BasicPoolsContext } from "../providers/BasicPoolsProvider"
+import { TokensContext } from "../providers/TokensProvider"
+import { useContext } from "react"
+
+export const useTokenMaps = (): {
```

Comment:
You can simplify this whole file once https://github.com/saddle-finance/saddle-frontend/pull/1193 lands